service/src/test/java/bio/terra/externalcreds/dataAccess/LinkedAccountDAOTest.java

```diff
@@ -39,7 +49,7 @@ void testInsertAndGetLinkedAccount() {
 
     var loadedLinkedAccount =
         linkedAccountDAO.getLinkedAccount(linkedAccount.getUserId(), linkedAccount.getProviderId());
-    assertEquals(Optional.of(savedLinkedAccount), loadedLinkedAccount);
+    assertEquals(savedLinkedAccount, loadedLinkedAccount.get());
```

Comment:
this (and the next) was better as it was written before

Reply:
Cool, I'll put those back. I thought .get() might be neater but that makes sense!